src/test/run-make/compiler-lookup-paths/Makefile

```diff
@@ -21,7 +21,7 @@ all: $(TMPDIR)/libnative.a
 	mkdir -p $(TMPDIR)/e1
 	mkdir -p $(TMPDIR)/e2
 	$(RUSTC) e.rs -o $(TMPDIR)/e1/libe.rlib
-	$(RUSTC) e.rs -o $(TMPDIR)/e2/libe.rlib
+	$(RUSTC) e.rs -o $(TMPDIR)/e2/libe-alt.rlib
```

Comment:
Hm I'm a little confused here, don't both these libraries have the same SVH? I don't follow why changing the filename causes them to suddenly be "different"